AuthenticationSDK/spec/MerchantConfigData.rb

```diff
@@ -0,0 +1,59 @@
+public
+class MerchantConfigData
+    def merchantConfigProp()
+        merchantId='testrest'
+        runEnvironment='cybersource.environment.sandbox'
+        timeout=1000 #In Milliseconds
+        authenticationType='JWT' #'HTTP_Signature'
+        logDirectory='../../../AuthenticationSDK/src/log'
+        logDirectory=File.expand_path(logDirectory,__FILE__)
+        jsonFilePath='./PostRequestData.json'
+        logSize=10485760
+        enableLog=true
+        proxyAddress='userproxy.visa.com'
```

Comment:
Remove the hardcoded value and get this address from environment variable.
OR
Comment this line out and put dummy value in place of the actual proxy.